sgf-viewer/prepare_data/prepare_data.py

```diff
@@ -0,0 +1,98 @@
+import subprocess
+import json
+import re
+import csv
+from pathlib import Path
+import sys
+
+mypath = Path(__file__).parent.parent.parent / "parsing_server"
+sys.path.append(str(mypath))
```

Comment:
Do we really need to mess with `sys.path`? Can we not just install `parsing_server` as a package?

Reply:
Moved game_info.py into package